README.md

```diff
@@ -5,4 +5,78 @@
+and `github.com/amdonov/xmlsig`.
+
+## Basic Overview
+
+The most important packages of the library:
+<pre>
+/pkg
+    /provider definitions and implementation of a SAML provider (Identity provider)
+        /serviceprovider definitions and implementation of a SAML user (Service provider)
+        /xml definitions of SAML xml messages
+        /checker helper to abstract the SAML standard in the processes
+        /signature implementation to handle and create SAML signature
+</pre>
+
+## Features
+
+|                   | POST-binding | Redirect-binding | Artifact-binding | Request signing | Response signing | Metadata signing | Response encryption | Assertion Query/Request | Attribute Query | NameID Mapping |
+|-------------------|--------------|------------------|------------------|-----------------|------------------|---------|--------------------|-------------------------|-----------------|----------------|
+| Identity provider | yes          | yes              | no               | yes             | yes              | yes              | no                | no                      | yes             | no             |
+
+## Resources
+
+For your convenience you can find the relevant standards linked below.
+
+- [Security Assertion Markup Language (SAML) V2.0 Technical Overview](http://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0.html)
+- [Assertions and Protocols for the OASIS Security Assertion Markup Language (SAML) V2.0 – Errata Composite](https://www.oasis-open.org/committees/download.php/35711/sstc-saml-core-errata-2.0-wd-06-diff.pdf)
+- [Bindings for the OASIS Security Assertion Markup Language (SAML) V2.0 – Errata Composite](https://www.oasis-open.org/committees/download.php/35387/sstc-saml-bindings-errata-2.0-wd-05-diff.pdf)
+- [Profiles for the OASIS Security Assertion Markup Language (SAML) V2.0 – Errata Composite](https://www.oasis-open.org/committees/download.php/35389/sstc-saml-profiles-errata-2.0-wd-06-diff.pdf)
+- [Metadata for the OASIS Security Assertion Markup Language (SAML) V2.0 – Errata Composite](https://www.oasis-open.org/committees/download.php/35391/sstc-saml-metadata-errata-2.0-wd-04-diff.pdf)
+- [Conformance Requirements for the OASIS Security Assertion Markup Language (SAML) V2.0 – Errata Composite](https://www.oasis-open.org/committees/download.php/35393/sstc-saml-conformance-errata-2.0-wd-04-diff.pdf)
+- [xml.com SAML Specifications](http://saml.xml.org/saml-specifications)
+- [SAML Developer Tools from Onelogin](https://www.samltool.com/online_tools.php)
+
+## Supported Go Versions
+
+For security reasons, we only support and recommend the use of one of the latest two Go versions (:white_check_mark:).  
+Versions that also build are marked with :warning:.
+
+| Version | Supported          |
+|---------|--------------------|
+| <1.15   | :x:                |
+| 1.15    | :warning:          |
+| 1.16    | :warning:          |
+| 1.17    | :white_check_mark: |
```

Comment:
text recommends to use only the latest two. You've marked the latest three.

Reply:
added 1.19 after i copied in the table, corrected